pycbc/frame/frame.py

```diff
@@ -631,7 +631,8 @@ def __init__(self, frame_src,
                        max_buffer=2048,
                        valid_mask=3,
                        force_update_cache=False,
-                       increment_update_cache=None):
+                       increment_update_cache=None,
+                       padding=0):
```

Comment:
Hmmm, maybe just keep this in the LiveStrainBuffer class? If we aren't using it here, it seems a bit odd to me.